modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/api/construction/ConstructionPayloads.scala

```diff
@@ -0,0 +1,51 @@
+package org.tessellation.rosetta.domain.api.construction
+
+import org.tessellation.ext.derevo.magnoliaCustomizable.snakeCaseConfiguration
+import org.tessellation.rosetta.domain.operation.Operation
+import org.tessellation.rosetta.domain.{NetworkIdentifier, RosettaPublicKey, SigningPayload}
+import org.tessellation.security.hex.Hex
+
+import derevo.cats.{eqv, show}
+import derevo.circe.magnolia.{customizableDecoder, customizableEncoder}
+import derevo.derive
+
+case object ConstructionPayloads {
+
+  @derive(customizableDecoder)
+  case class Request(
+    networkIdentifier: NetworkIdentifier,
+    operations: List[Operation],
+//    metadata: Option[RequestMetadata],   // TODO delete if we don't need it
+    publicKeys: Option[List[RosettaPublicKey]]
```

Comment:
Is there a point in using `Option[List[...]]` when a list can be simply empty? Maybe it makes sens to have `Option[NonEmptyList[...]]`